lib/list_validator.rb

```diff
@@ -0,0 +1,11 @@
+class ListValidator
+  def initialize(list)
+    @list = list
```

Comment:
Yeah, it's a nitpick - I don't think we'll be handling anything huge, so fine to merge.